.github/workflows/build-publish-develop-pr.yml

```diff
@@ -120,6 +152,8 @@ jobs:
 
   image-tag:
     runs-on: ubuntu-latest
+    needs: changes
+    if: ${{ (github.event_name != 'pull_request' && github.event_name != 'merge_group') || needs.changes.outputs.not_ignored_changes == 'true' }}
```

Comment:
`(github.event_name != 'pull_request' && github.event_name != 'merge_group') || needs.changes.outputs.not_ignored_changes == 'true'` is repeated 12 times. Is there a way to use a single well named variable instead?

Reply:
@jmank88 we could have an `output` in `changes` job named something like `should_run_tests`. Would that work?

Reply:
cc @HenryNguyen5 WDYT?

Reply:
Yeah that would work, that way we don't have to update so many references each time

Reply:
@jmank88, @HenryNguyen5 I fixed it in 670d77880f6fe22c50942dd58de6d47cd0be9435 according to your feedback. Can you check and resolve this conversation?